app/src/main/java/org/oppia/app/onBoarding/OnBoardingActivityPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.app.onBoarding
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.on_boarding_activity.*
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.app.profile.ProfileActivity
+import org.oppia.domain.OnBoardingFlowController
+import javax.inject.Inject
+
+/** The presenter for [OnBoardingingActivity]. */
+@ActivityScope
+class OnBoardingActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val onBoardingFlowController: OnBoardingFlowController
+) {
+
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.on_boarding_activity)
+    activity.onBoard.setOnClickListener({ subscribeOnBoarding() })
```

Comment:
Update to `activity.onBoard.setOnClickListener { subscribeOnBoarding() }`

Reply:
updated to     activity.onBoard.setOnClickListener { subscribeOnBoarding() }